lib/plug/mailbox_preview.ex

```diff
@@ -37,7 +37,7 @@ if Code.ensure_loaded?(Plug) do
     def call(conn, opts) do
       conn =
         conn
-        |> assign(:base_path, opts[:base_path] || "/")
+        |> assign(:base_path, opts[:base_path] || Enum.join(["/" | conn.script_name]))
```

Comment:
Shouldn't this be `Path.join` instead of `Enum.join`?

Reply:
True, that's what went through my head but didn't make it into the code.
Code review wins :)